src/lib.rs

```diff
@@ -32,9 +32,13 @@ pub mod bundle;
 #[cfg(feature = "circuit")]
 pub mod circuit;
 mod constants;
+pub mod issuance;
+pub mod issuance_auth;
+pub mod issuance_sighash_versioning;
 pub mod keys;
 pub mod note;
-pub mod note_encryption;
+pub mod orchard_flavor;
+pub mod orchard_sighash_versioning;
```

Comment:
str4d ([Link](https://github.com/zcash/orchard/pull/471#discussion_r2542392826)): Remove the redundant module prefixes; these are already in the `orchard` crate namespace, and `orchard::orchard_signash_versioning::OrchardSighashVersion` is unnecessary duplicative.
```suggestion
pub mod flavor;
pub mod sighash_versioning;
```
(or `protocol_flavor` if `orchard::flavor` is not sufficiently clear).

Also, document the addition of these modules in the changelog.